src/slack/utilities/getDashboardContext.ts

```diff
@@ -0,0 +1,16 @@
+import { Subscriber } from '../../entities/subscriber';
+import { Config } from '../../entities/config';
+
+interface DashboardContext {
+  [key: string]: boolean;
+}
+
+export async function getDashboardContext(userId: string): Promise<DashboardContext> {
+  const teamRegistrationToggle = (await Config.findOne({ key: 'teamRegistrationActive' })) || { key: 'teamRegistrationActive', value: 'false' };
+  const subscriber = await Subscriber.findOne({ slackId: userId, isActive: true });
+
+  return {
+    [teamRegistrationToggle.key]: teamRegistrationToggle.value === 'true',
+    isSubscribed: Boolean(subscriber),
```

Comment:
Instead of getting the whole subscriber and casting it to a boolean, why don't we just get the count and then `isSubscribed` could be `count === 1`. It would also be a faster query.